x-pack/plugins/security_solution/public/timelines/components/graph_overlay/index.test.tsx

```diff
@@ -54,20 +72,18 @@ describe('GraphOverlay', () => {
   });
 
   describe('when used in an events viewer (i.e. in the Detections view, or the Host > Events view)', () => {
-    test('it has 100% width when NOT in full screen mode', async () => {
-      const wrapper = mount(
+    test('it has 100% width when NOT in full screen mode', () => {
+      const wrapper = render(
         <TestProviders>
-          <GraphOverlay timelineId={TimelineId.test} />
+          <GraphOverlay timelineId={TimelineId.test} SessionView={<div />} Navigation={<div />} />
```

Comment:
I would set a`mockSessionView` and `mockNavigation` so it's easier to change in the future